pyeudiw/openid4vp/presentation_submission/config.yml

```
@@ -0,0 +1,15 @@
+formats:
+  - name: "dc+sd-jwt"
+    module: "pyeudiw.openid4vp.presentation_submission"
+    class: "VcSdJwt"
```

https://github.com/italia/eudi-wallet-it-python/blob/dev/pyeudiw/sd_jwt/verifier.py

```suggestion
    module: "pyeudiw.sd_jwt.verifier"
    class: "SDJWTVerifier"
```